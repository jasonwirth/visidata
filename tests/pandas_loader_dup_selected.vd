sheet	col	row	longname	input	keystrokes	comment
global		filetype	set-option	pandas		
			open-file	sample_data/sample.tsv	o	
sample		0	select-row		s	select current row
sample		1	select-row		s	select current row
sample		2	select-row		s	select current row
sample			dup-selected		"	open duplicate sheet with only selected rows
