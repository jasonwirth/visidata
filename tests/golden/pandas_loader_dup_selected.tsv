OrderDate	Region	Rep	Item	Units	Unit_Cost	Total
2016-01-06	East	Jones	Pencil	95	1.99	189.05
2016-01-23	Central	Kivell	Binder	50	19.99	999.50
2016-02-09	Central	Jardine	Pencil	36	4.99	179.64
